doc/sample-nginx.conf

```
@@ -0,0 +1,62 @@
+worker_processes  1;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    # Blockdaemon with bearer token authentication
+    server {
+        listen 7900;
+
+        location / {
+            proxy_pass https://yourserver-solana-01.bdnodes.net;
+            proxy_http_version 1.1;
+            proxy_set_header Host $proxy_host;
+            proxy_set_header Authorization "Bearer ReplaceMeWithYourAuthToken";
+            proxy_redirect https://$proxy_host http://$server_name:$server_port;
+            proxy_redirect https:// http://;
+        }
+    }
+
+    # Figment RPC with auth
+    server {
+        listen 7901;
+
+        location / {
+            proxy_pass https://solana--mainnet--rpc.datahub.figment.io/apikey/ReplaceMeWithYourAuthToken;
```

Just today (Wed Sept 1), Figment released a new endpoint which combines WS and RPC into a single domain. You can now use `solana--mainnet.datahub.figment.io` for both WS and RPC. We do plan to deprecate the WS and RPC-specific endpoints at some point in the future. So, it would be ideal to use the single domain.

```suggestion
            proxy_pass https://solana--mainnet.datahub.figment.io/apikey/ReplaceMeWithYourAuthToken;
```